hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -999,6 +1171,402 @@ set_false_path -from [get_clocks SPI_DEV_FAST_PASS_CSB_CLK] \
+set_multicycle_path -setup -start 1 \
+    -from [get_clocks IO_DIV2_CLK] \
+    -to [get_clocks SPI_HOST1_CLK]
+set_multicycle_path -hold -start 1 \
+    -from [get_clocks IO_DIV2_CLK] \
+    -to [get_clocks SPI_HOST1_CLK]
+
+# set multicycle path for data going from SPI_HOST1_CLK to logic
+# the SPI host logic will read these paths at "full cycle"
+set_multicycle_path -setup -end 2 \
+    -from [get_clocks SPI_HOST1_CLK] \
+    -to [get_clocks IO_DIV2_CLK]
+set_multicycle_path -hold -end 1 \
+    -from [get_clocks SPI_HOST1_CLK] \
+    -to [get_clocks IO_DIV2_CLK]
+
+# 3. Adjust the asynchronous clock groups so SPI_HOST1_CLK is grouped with IO_DIV2_CLK.
+
+# -    -group [get_clocks IO_DIV2_CLK                                  ] \
+# +    -group [get_clocks {IO_DIV2_CLK SPI_HOST1_CLK}                  ] \
+
+# Approved by Ziv
+#  SPI_HOST_D0
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_9__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#  SPI_HOST_D1
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_10__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#  SPI_HOST_D2
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_11__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#  SPI_HOST_D3
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_12__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#  SPI_DEV_D0
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_15__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#  SPI_DEV_D1
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_16__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#  SPI_DEV_D2
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_17__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#  SPI_DEV_D3
+set_false_path -hold -fall_through [get_pins u_padring/gen_dio_pads_18__u_dio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#IOB0
+set_false_path -hold -fall_through [get_pins u_padring/gen_mio_pads_9__u_mio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#IOB1
+set_false_path -hold -fall_through [get_pins u_padring/gen_mio_pads_10__u_mio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#IOB2
+set_false_path -hold -fall_through [get_pins u_padring/gen_mio_pads_11__u_mio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+#IOB3
+set_false_path -hold -fall_through [get_pins u_padring/gen_mio_pads_12__u_mio_pad/gen_techlib_u_impl_techlib/gen_bidir_u_pad_macro_PBIDIR_33_33_FS_DR/OE]
+
+set_false_path  -from IO_DIV2_CLK -through [get_cells -hierarchical -filter "full_name =~ *u_spi_host1*"] -to IOA0
```

Comment:
Nit. duplicates. The sections L1311-1353 and L1355-1397 are identical?

Reply:
Thanks for flagging this @hcallahan-lowrisc . I'll remove the duplicate in the next push.